CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+cmake_minimum_required(VERSION 3.20...3.29)
+# 3.20 is the minimum requirement for LLVM
+# https://www.llvm.org/docs/CMake.html#quick-start
+
+project(
+        ${SKBUILD_PROJECT_NAME}
+	VERSION ${SKBUILD_PROJECT_VERSION}
+	LANGUAGES CXX)
+
+set(
+	CMAKE_CXX_STANDARD
+	14
+	CACHE STRING "The C++ standard to compile with, 14+")
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+# Linking to OpenMP will be outside the source tree, so include it in INSTALL_RPATH anyway
+# Repair steps will bundle it as needed.
+set(CMAKE_INSTALL_RPATH_USE_LINK_PATH ON)
+
+find_package(
+	OpenMP
+	REQUIRED)
+
+find_package(
+	Python
+	COMPONENTS Interpreter Development.Module NumPy
+	REQUIRED)
+
+set(UseSWIG_MODULE_VERSION 2)
+find_package(SWIG REQUIRED)
+include(UseSWIG)
+
+message(STATUS "Working directory is ${CMAKE_CURRENT_SOURCE_DIR}")
+
+set_property(SOURCE "wasserstein/swig/wasserstein.i" PROPERTY CPLUSPLUS ON)
+set_property(SOURCE "wasserstein/swig/wasserstein.i" PROPERTY SWIG_MODULE_NAME wasserstein)
+swig_add_library(
+	wasserstein
+        TYPE MODULE
+	LANGUAGE python
+	SOURCES "wasserstein/swig/wasserstein.i")
```

Comment:
Given this we can also remove the `src/wasserstein/wasserstein.py` file as it is generated from SWIG.